java/src/jmri/jmrit/vsdecoder/VSDecoderManager.java

```diff
@@ -761,8 +761,8 @@ public void reporterPropertyChange(PropertyChangeEvent event) {
             PhysicalLocationReporter arp = (PhysicalLocationReporter) event.getSource();
             // Need to decide which reporter it is, so we can use different methods
             // to extract the address and the location.
-            if (event.getNewValue() instanceof String) {
-                String newValue = (String) event.getNewValue();
+            if (event.getNewValue() instanceof jmri.Reportable) {
+                String newValue = ((jmri.Reportable) event.getNewValue()).toReportString();
```

Comment:
Just looking at what follows in the source file here, This isn't quite what you want.

The else part of this If statement reads:
https://github.com/JMRI/JMRI/blob/a383810d039fec59787a82af194e495a54790718/java/src/jmri/jmrit/vsdecoder/VSDecoderManager.java#L769

But all the concrete implementations of IDTag also implement the Reportable interface, so the else if will never execute.

The TranspondingTag, which replaced the String that was previously reported by the LnReporter, really should just be handled like any other IDTag.

Reply:
@pabender Thanks!  Your help is greatly appreciated.
Will prepare a change.